exercises/ocr-numbers/Example.fs

```diff
@@ -0,0 +1,105 @@
+module OcrNumbers
+
+let characterWidth = 3
+let characterHeight = 4
+
+let digitPatterns = 
+    Map.ofList 
+        [(
+          [|" _ ";
+            "| |";
+            "|_|"; 
+            "   "|],
+            "0"
+        );
+        (
+          [|"   ";
+            "  |";
+            "  |";
+            "   "|],
+            "1"
+        );
+        (
+          [|" _ ";
+            " _|";
+            "|_ ";
+            "   "|],
+            "2"
+        );
+        (
+          [|" _ ";
+            " _|";
+            " _|";
+            "   "|],
+            "3"
+        );
+        (
+          [|"   ";
+            "|_|";
+            "  |";
+            "   "|],
+            "4"
+        );
+        (
+          [|" _ ";
+            "|_ ";
+            " _|";
+            "   "|],
+            "5"
+        );
+        (
+          [|" _ ";
+            "|_ ";
+            "|_|";
+            "   "|],
+            "6"
+        );
+        (
+          [|" _ ";
+            "  |";
+            "  |";
+            "   "|],
+            "7"
+        );
+        (
+          [|" _ ";
+            "|_|";
+            "|_|";
+            "   "|],
+            "8"
+        );
+        (
+          [|" _ ";
+            "|_|";
+            " _|";
+            "   "|],
+            "9"
+        )]
+
+let matchCharacter input = 
+    match Map.tryFind input digitPatterns with
+    | Some x -> x
+    | None -> "?"
+
+let rows (input:string) = 
+    input.Split('\n')
+    |> Array.chunkBySize characterHeight
+
+let rowToCharacters (row: string[]) = 
```

Comment:
Absolutely! Sometimes I find myself just trying things out, removing types, to see if it still compiles, and often it still does!